### PR TITLE
UML for income and expense

### LINKED ISSUE #109: Add income and expense to DG

### PR DESCRIPTION
**Pull request by NeoMinWei**
fixes #109

### CODE REVIEW

**Review by ryan1604 — APPROVED** [review]
LGTM, line 8 of CashFlowClassDiagram has ] instead of } for abstract

**Review by wwweert123 — APPROVED** [review]
Nice! Its good that you add the multiplicity as well! Mayeb you can show the example output as well?

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1,22 +1,82 @@
...
+            cashflowList.addIncome(amount, incomeType, recur);
+            break;
+        case EXPENSE:
+            cashflowList.addExpense(amount, expenseType, recur);
+            break;
+        default:
+            ui.showMessage("Unidentified entry.");
+            break;
+        }
+```
```
Comment (wwweert123) [inline docs/DeveloperGuide.md:65]: Maybe can include a screenshot of an example output here!

**Review by wwweert123 — COMMENTED** [review]
Goodjob!

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (NeoMinWei) [reply docs/DeveloperGuide.md:65]: very good suggestion, will consider

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1,22 +1,82 @@
...
-[Addressbook-level3](https://github.com/se-edu/addressbook-level3)
+- **round() method in Cashflow.java**
+  - author: mhadidg
+  - source: https://stackoverflow.com/questions/2808535/round-a-double-to-2-decimal-places
+- **capitalize() method in Cashflow.java**
+  - author: Nick Bolton
+  - source: https://stackoverflow.com/questions/1892765/how-to-capitalize-the-first-character-of-each-word-in-a-string
+    
+- **DG adapted from**
+  - [Addressbook-level3](https://github.com/se-edu/addressbook-level3)
```
Comment (ryan1604) [inline docs/DeveloperGuide.md:12]: Next time can change to use * instead of - to follow markdown coding standard

**Pull request merged**